src/main/resources/templates/javascript/exercise/src/mergesort.js

```diff
@@ -0,0 +1,3 @@
+export default class MergeSort {
+    // TODO: implement in performSort(Date[])
```

Comment:
In the Rust template (and the Java template), we have method bodies in the template for `performSort` / `perform_sort`. Is there any particular reason you don't add a `performSort(input)` here?

Reply:
Yes, this is intentional. The idea is to leave as much as possible as an exercise.

In compiled languages like Java and Rust the method definition needs to be present to use them in tests without build failures. If build failures happen, no test results are reported. You can get around build failures using Reflection or Macros as done as in the other tests. The `performSort` tests are simpler, however, and expect the correct definition.

JavaScript (and Python) don't have to do this, because they can detect the missing definition at runtime.

Reply:
Makes sense, thanks 👍